[2022-12-03 02:39:27,317] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.call_dataset_task manual__2022-12-03T02:39:24.814374+00:00 [queued]>
[2022-12-03 02:39:27,331] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.call_dataset_task manual__2022-12-03T02:39:24.814374+00:00 [queued]>
[2022-12-03 02:39:27,331] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 02:39:27,331] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-03 02:39:27,332] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 02:39:27,346] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): call_dataset_task> on 2022-12-03 02:39:24.814374+00:00
[2022-12-03 02:39:27,354] {standard_task_runner.py:52} INFO - Started process 176 to run task
[2022-12-03 02:39:27,361] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'call_dataset_task', 'manual__2022-12-03T02:39:24.814374+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpzqc1tz5s', '--error-file', '/tmp/tmpspy52sge']
[2022-12-03 02:39:27,363] {standard_task_runner.py:77} INFO - Job 46: Subtask call_dataset_task
[2022-12-03 02:39:27,446] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.call_dataset_task manual__2022-12-03T02:39:24.814374+00:00 [running]> on host c87e816ea86f
[2022-12-03 02:39:27,556] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=call_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T02:39:24.814374+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-03T02:39:24.814374+00:00
[2022-12-03 02:39:27,702] {python.py:175} INFO - Done. Returned value was: None
[2022-12-03 02:39:27,716] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=call_dataset_task, execution_date=20221203T023924, start_date=20221203T023927, end_date=20221203T023927
[2022-12-03 02:39:27,775] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-03 02:39:27,829] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
